### PR TITLE
Aj/data bug

### PR DESCRIPTION
**Pull request by staubina**

### CODE REVIEW

File: src/users/EnrollmentForm.jsx [hunk 1]
```diff
@@ -95,11 +98,15 @@ export default function EnrollmentForm({
...
             onChange={(event) => setComments(event.target.value)}
+            ref={forwardedRef}
           />
         </div>
         <div>
           <Button
-            className="btn-primary mr-3"
+            className={classNames(
+                "btn-primary mr-3",
+                reason ? '' : 'disabled'
```
Comment (tuchfarber) [inline src/users/EnrollmentForm.jsx]: I think the preferred way to do this is `{'disabled': !!reason}` https://www.npmjs.com/package/classnames#usage

File: src/users/EntitlementForm.jsx [hunk 2]
```diff
@@ -100,11 +103,15 @@ export default function EntitlementForm({
...
             onChange={(event) => setComments(event.target.value)}
+            ref={forwardedRef}
           />
         </div>
         <div>
           <Button
-            className="btn-primary mr-3"
+            className={classNames(
+              "btn-primary mr-3",
+              courseUuid && mode && comments ? '' : 'disabled'
```
Comment (tuchfarber) [inline src/users/EntitlementForm.jsx]: Same as above

**Review by crice100 — APPROVED** [review]
A ticket number and/or description of the changes might be nice.

**Review by tuchfarber — COMMENTED** [review]
Just two small nits since it was approved.

**Comment by staubina** [comment]
@tuchfarber I made the requested changes.

**Pull request merged**